packages/plugin-react-native-unhandled-rejection/rejection-handler.js

```diff
@@ -3,19 +3,29 @@
 */
 
 const rnPromise = require('promise/setimmediate/rejection-tracking')
-const createReportFromErr = require('@bugsnag/core/lib/report-from-error')
+const ensureError = require('@bugsnag/core/lib/ensure-error')
 
 module.exports = {
   init: (client) => {
-    if (!client.config.autoNotify) return () => {}
+    if (client._config.autoDetectErrors === false || client._config.autoDetectUnhandledRejections === false) return () => {}
```

Comment:
The spec states that setting a more granular `autoDetect` flag to true will override the behaviour of `autoDetectErrors` - the plugins or spec will need to be updated accordingly

Reply:
The implementation is correct, the spec needs updating.